--- node_modules/@tanstack/start-plugin-core/dist/esm/vite/prerender.js	2026-07-25 14:28:39
+++ node_modules/@tanstack/start-plugin-core/dist/esm/vite/prerender.js.new	2026-07-25 16:05:06
@@ -11,7 +11,7 @@
 	process.env.TSS_CLIENT_OUTPUT_DIR = outputDir;
 	const previewServer = await startPreviewServer(serverEnv.config);
 	const baseUrl = getResolvedUrl(previewServer);
-	return prerender({
+	const p = prerender({
 		startConfig,
 		handler: {
 			getClientOutputDirectory() {
@@ -26,6 +26,9 @@
 			}
 		}
 	});
+	return p.finally(() => {
+		try { import('child_process').then(cp => cp.execSync("pkill -f 'wrangler pages dev'")) } catch (e) {}
+	});
 }
 async function startPreviewServer(viteConfig) {
 	const vite = await import("vite");
