--- a/dist/_chunks/nitro2.mjs
+++ b/dist/_chunks/nitro2.mjs
@@ -81,7 +81,7 @@
 		...args,
 		"--port",
 		String(randomPort),
-		"--host",
+		"--ip",
 		"localhost"
 	], {
 		stdio: "inherit",
@@ -121,6 +121,7 @@
 			}, req, socket, head, opts);
 		},
 		async close() {
+			import('child_process').then(cp => cp.execSync("pkill -f 'wrangler pages dev'")).catch(()=>{});
 			killChild("SIGTERM");
 		}
 	};
