--- a/dist/_chunks/nitro2.mjs
+++ b/dist/_chunks/nitro2.mjs
@@ -78,10 +78,8 @@ async function runPreviewCommand(opts) {
 	const { getRandomPort, waitForPort } = await import("get-port-please");
 	const randomPort = await getRandomPort();
 	const child = spawn(arg0, [
-		...args,
-		"--port",
-		String(randomPort),
-		"--ip",
-		"localhost"
+		...args,
+		"--port",
+		String(randomPort)
 	], {
 		stdio: "inherit",
 		cwd: opts.rootDir,
